app/scripts/lib/metrics.js

```diff
@@ -339,6 +339,8 @@ define([
      */
     logScreen: function (screenName) {
       this.logEvent(this.screenToId(screenName));
+      // log the path as a separate path event
+      this.logEvent('path.' + this._window.location.pathname);
```

Comment:
I was just checking, a second problem is that screenName for any panel in the iframe flow is always `settings`.

```
["screen.settings","path./","screen.settings","path./","screen.settings","path./","screen.settings","path./","screen.settings.avatar.change","path./","settings.avatar.change.profile_image_not_shown","screen.settings","path./"]
```

Filed #3029, as I noted there, not a problem if we banish the the settings page from being iframed (#2945)